### PR TITLE
remove some iota.go/v3 dependency

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: packages/parameters/l1parameters.go [hunk 1]
```diff
@@ -58,18 +49,6 @@ var (
 	L1ForTesting = &L1Params{
 		// There are no limits on how big from a size perspective an essence can be, so it is just derived from 32KB - Message fields without payload = max size of the payload
 		MaxPayloadSize: MaxPayloadSize,
-		Protocol: &iotago.ProtocolParameters{
```
Comment (lmoe) [inline packages/parameters/l1parameters.go]: What about the ProtocolParams here?

I've thought they can change every Epoch, wouldnt it make sense to refactor this place here?

Comment (howjmay) [reply packages/parameters/l1parameters.go]: I thinking to refactor that in the following PR to make the incoming PR smaller. I can remove this change to make this PR contain less things

Comment (howjmay) [reply packages/parameters/l1parameters.go]: removed

Comment (lmoe) [reply packages/parameters/l1parameters.go]: Okay, both ways are fine. Just wanted to ask. :D

Comment (howjmay) [reply packages/parameters/l1parameters.go]: I just removed the mock_ledger too. Please check. Sorry

**Review by dessaya — APPROVED** [review]

**Pull request merged**